### DISCUSSION TITLE
Change font size per window.

### DISCUSSION THREAD

**Discussion by beholdenspore28** [kovidgoyal/kitty#6856] [Ideas]
Let's say you open kitty. You hit alt+enter to split a few new windows open. You open a text editor, a task manager, and a debugger. Wouldn't it be nice if you could enable an option to adjust the font size for each new split individually? I write code every day using kitty. I think this would be a really nice feature cause not every app's font size needs to be as big as my code editor.

**Comment by trathborne** [comment]
Yes! There's 'current' for just this window, 'all' for all OS windows ... and there should be 'pane' for just the current pane and how about 'tab' for just the current tab?
